toolbox/pkg_check/main.go

```diff
@@ -26,20 +26,24 @@ import (
 	"strconv"
 	"strings"
 
+	"sort"
+
 	"cloud.google.com/go/storage"
 	"golang.org/x/net/context"
 )
 
 const (
-	jobName      = "JOB_NAME"
-	buildID      = "BUILD_ID"
-	defaultValue = "Default"
+	jobName         = "JOB_NAME"
+	buildID         = "BUILD_ID"
+	defaultValue    = "Default"
+	defaultTreshold = 80.0
```

Comment:
Maybe we should get the default from a flag

Reply:
done